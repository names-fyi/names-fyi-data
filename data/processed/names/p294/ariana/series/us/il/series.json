[{"y":1969,"t":6,"m":0,"f":6,"mr":null,"fr":987,"mp":0,"fp":0.00007076640011322624},{"y":1973,"t":5,"m":0,"f":5,"mr":null,"fr":1166,"mp":0,"fp":0.00007120681306787434},{"y":1976,"t":5,"m":0,"f":5,"mr":null,"fr":1213,"mp":0,"fp":0.0000715195033685686},{"y":1978,"t":8,"m":0,"f":8,"mr":null,"fr":898,"mp":0,"fp":0.00011281906642222535},{"y":1979,"t":9,"m":0,"f":9,"mr":null,"fr":898,"mp":0,"fp":0.00012047870204278333},{"y":1980,"t":14,"m":0,"f":14,"mr":null,"fr":658,"mp":0,"fp":0.00018112660749864155},{"y":1981,"t":30,"m":0,"f":30,"mr":null,"fr":375,"mp":0,"fp":0.0003991962848132427},{"y":1982,"t":21,"m":0,"f":21,"mr":null,"fr":471,"mp":0,"fp":0.0002821822090835797},{"y":1983,"t":21,"m":0,"f":21,"mr":null,"fr":461,"mp":0,"fp":0.00028853684340693315},{"y":1984,"t":11,"m":0,"f":11,"mr":null,"fr":730,"mp":0,"fp":0.0001514880255601614},{"y":1985,"t":16,"m":0,"f":16,"mr":null,"fr":564,"mp":0,"fp":0.00021917207747732938},{"y":1986,"t":17,"m":0,"f":17,"mr":null,"fr":537,"mp":0,"fp":0.00023965940169735247},{"y":1987,"t":39,"m":0,"f":39,"mr":null,"fr":294,"mp":0,"fp":0.0005374121537825547},{"y":1988,"t":33,"m":0,"f":33,"mr":null,"fr":336,"mp":0,"fp":0.00044776119402985075},{"y":1989,"t":62,"m":0,"f":62,"mr":null,"fr":211,"mp":0,"fp":0.0008021009871016986},{"y":1990,"t":70,"m":0,"f":70,"mr":null,"fr":195,"mp":0,"fp":0.0008917197452229299},{"y":1991,"t":77,"m":0,"f":77,"mr":null,"fr":178,"mp":0,"fp":0.0009932023682072053},{"y":1992,"t":81,"m":0,"f":81,"mr":null,"fr":177,"mp":0,"fp":0.001076011583729642},{"y":1993,"t":74,"m":0,"f":74,"mr":null,"fr":188,"mp":0,"fp":0.000984225786715612},{"y":1994,"t":110,"m":0,"f":110,"mr":null,"fr":143,"mp":0,"fp":0.0014818207535732086},{"y":1995,"t":90,"m":0,"f":90,"mr":null,"fr":170,"mp":0,"fp":0.0012404211919069409},{"y":1996,"t":118,"m":0,"f":118,"mr":null,"fr":132,"mp":0,"fp":0.0016516201273707048},{"y":1997,"t":153,"m":0,"f":153,"mr":null,"fr":104,"mp":0,"fp":0.0021884968030781993},{"y":1998,"t":139,"m":0,"f":139,"mr":null,"fr":117,"mp":0,"fp":0.001974095324660569},{"y":1999,"t":126,"m":0,"f":126,"mr":null,"fr":120,"mp":0,"fp":0.0018044854352247013},{"y":2000,"t":190,"m":0,"f":190,"mr":null,"fr":73,"mp":0,"fp":0.002667228188390538},{"y":2001,"t":156,"m":0,"f":156,"mr":null,"fr":89,"mp":0,"fp":0.002216916780354707},{"y":2002,"t":197,"m":0,"f":197,"mr":null,"fr":73,"mp":0,"fp":0.0028686674530019078},{"y":2003,"t":223,"m":0,"f":223,"mr":null,"fr":59,"mp":0,"fp":0.003218868632630379},{"y":2004,"t":226,"m":0,"f":226,"mr":null,"fr":56,"mp":0,"fp":0.003300089073200648},{"y":2005,"t":233,"m":0,"f":233,"mr":null,"fr":50,"mp":0,"fp":0.0034699469827842973},{"y":2006,"t":206,"m":0,"f":206,"mr":null,"fr":67,"mp":0,"fp":0.0030234981580144716},{"y":2007,"t":196,"m":0,"f":196,"mr":null,"fr":66,"mp":0,"fp":0.002895211084522438},{"y":2008,"t":185,"m":0,"f":185,"mr":null,"fr":76,"mp":0,"fp":0.0027946463639384874},{"y":2009,"t":176,"m":0,"f":176,"mr":null,"fr":69,"mp":0,"fp":0.00278414933164597},{"y":2010,"t":160,"m":0,"f":160,"mr":null,"fr":75,"mp":0,"fp":0.002632228345808999},{"y":2011,"t":145,"m":0,"f":145,"mr":null,"fr":76,"mp":0,"fp":0.0024395168074295903},{"y":2012,"t":135,"m":0,"f":135,"mr":null,"fr":85,"mp":0,"fp":0.002299242101677595},{"y":2013,"t":175,"m":0,"f":175,"mr":null,"fr":55,"mp":0,"fp":0.00302208713972404},{"y":2014,"t":210,"m":0,"f":210,"mr":null,"fr":40,"mp":0,"fp":0.003565365025466893},{"y":2015,"t":225,"m":0,"f":225,"mr":null,"fr":37,"mp":0,"fp":0.0038478640079351507},{"y":2016,"t":190,"m":0,"f":190,"mr":null,"fr":50,"mp":0,"fp":0.0033007887147771097},{"y":2017,"t":177,"m":0,"f":177,"mr":null,"fr":52,"mp":0,"fp":0.0031914893617021275},{"y":2018,"t":146,"m":0,"f":146,"mr":null,"fr":62,"mp":0,"fp":0.0027319848056735463},{"y":2019,"t":148,"m":0,"f":148,"mr":null,"fr":59,"mp":0,"fp":0.002876411482323674},{"y":2020,"t":120,"m":0,"f":120,"mr":null,"fr":71,"mp":0,"fp":0.0024686278543509564},{"y":2021,"t":106,"m":0,"f":106,"mr":null,"fr":88,"mp":0,"fp":0.0021948441867688166},{"y":2022,"t":99,"m":0,"f":99,"mr":null,"fr":91,"mp":0,"fp":0.002127659574468085}]