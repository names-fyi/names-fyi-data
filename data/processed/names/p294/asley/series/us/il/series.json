[{"y":1987,"t":5,"m":0,"f":5,"mr":null,"fr":1299,"mp":0,"fp":0.00006889899407468652}]