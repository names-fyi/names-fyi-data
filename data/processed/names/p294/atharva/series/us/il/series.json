[{"y":2008,"t":7,"m":7,"f":0,"mr":1098,"fr":null,"mp":0.00009366177395399869,"fp":0},{"y":2009,"t":5,"m":5,"f":0,"mr":1398,"fr":null,"mp":0.00006894554680713173,"fp":0},{"y":2011,"t":6,"m":6,"f":0,"mr":1185,"fr":null,"mp":0.00008859357696567,"fp":0},{"y":2015,"t":7,"m":7,"f":0,"mr":1080,"fr":null,"mp":0.00010556954770989489,"fp":0},{"y":2022,"t":6,"m":6,"f":0,"mr":1162,"fr":null,"mp":0.00011639185257032008,"fp":0}]