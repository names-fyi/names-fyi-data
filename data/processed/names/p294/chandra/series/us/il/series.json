[{"y":1954,"t":5,"m":0,"f":5,"mr":null,"fr":897,"mp":0,"fp":0.00005016403639902481},{"y":1956,"t":7,"m":0,"f":7,"mr":null,"fr":795,"mp":0,"fp":0.00006671813494219351},{"y":1958,"t":16,"m":0,"f":16,"mr":null,"fr":529,"mp":0,"fp":0.00015063360259089798},{"y":1959,"t":12,"m":0,"f":12,"mr":null,"fr":638,"mp":0,"fp":0.00011074605928605707},{"y":1960,"t":11,"m":0,"f":11,"mr":null,"fr":679,"mp":0,"fp":0.00010221718363781664},{"y":1961,"t":5,"m":0,"f":5,"mr":null,"fr":1061,"mp":0,"fp":0.0000469051952194225},{"y":1962,"t":5,"m":0,"f":5,"mr":null,"fr":1076,"mp":0,"fp":0.00004860645299269931},{"y":1963,"t":9,"m":0,"f":9,"mr":null,"fr":743,"mp":0,"fp":0.00009080910916263911},{"y":1964,"t":10,"m":0,"f":10,"mr":null,"fr":727,"mp":0,"fp":0.0001018599629229735},{"y":1965,"t":15,"m":0,"f":15,"mr":null,"fr":558,"mp":0,"fp":0.0001643187345266525},{"y":1966,"t":19,"m":0,"f":19,"mr":null,"fr":486,"mp":0,"fp":0.00021596553644702592},{"y":1967,"t":28,"m":0,"f":28,"mr":null,"fr":397,"mp":0,"fp":0.0003276463291909476},{"y":1968,"t":26,"m":0,"f":26,"mr":null,"fr":414,"mp":0,"fp":0.0003107631626128011},{"y":1969,"t":42,"m":0,"f":42,"mr":null,"fr":306,"mp":0,"fp":0.0004953648007925837},{"y":1970,"t":35,"m":0,"f":35,"mr":null,"fr":362,"mp":0,"fp":0.0003989422331646378},{"y":1971,"t":36,"m":0,"f":36,"mr":null,"fr":354,"mp":0,"fp":0.0004354399758088902},{"y":1972,"t":46,"m":0,"f":46,"mr":null,"fr":288,"mp":0,"fp":0.0006166467820421733},{"y":1973,"t":32,"m":0,"f":32,"mr":null,"fr":348,"mp":0,"fp":0.0004557236036343957},{"y":1974,"t":47,"m":0,"f":47,"mr":null,"fr":259,"mp":0,"fp":0.0006741347408884235},{"y":1975,"t":35,"m":0,"f":35,"mr":null,"fr":323,"mp":0,"fp":0.0005024115755627009},{"y":1976,"t":30,"m":0,"f":30,"mr":null,"fr":372,"mp":0,"fp":0.00042911702021141166},{"y":1977,"t":24,"m":0,"f":24,"mr":null,"fr":441,"mp":0,"fp":0.0003318262889377411},{"y":1978,"t":29,"m":0,"f":29,"mr":null,"fr":371,"mp":0,"fp":0.0004089691157805669},{"y":1979,"t":31,"m":0,"f":31,"mr":null,"fr":369,"mp":0,"fp":0.0004149821959251426},{"y":1980,"t":30,"m":0,"f":30,"mr":null,"fr":381,"mp":0,"fp":0.0003881284446399462},{"y":1981,"t":19,"m":0,"f":19,"mr":null,"fr":516,"mp":0,"fp":0.0002528243137150537},{"y":1982,"t":22,"m":0,"f":22,"mr":null,"fr":458,"mp":0,"fp":0.0002956194571351787},{"y":1983,"t":14,"m":0,"f":14,"mr":null,"fr":609,"mp":0,"fp":0.00019235789560462208},{"y":1984,"t":19,"m":0,"f":19,"mr":null,"fr":494,"mp":0,"fp":0.0002616611350584606},{"y":1985,"t":13,"m":0,"f":13,"mr":null,"fr":660,"mp":0,"fp":0.00017807731295033013},{"y":1986,"t":12,"m":0,"f":12,"mr":null,"fr":697,"mp":0,"fp":0.00016917134237460173},{"y":1987,"t":15,"m":0,"f":15,"mr":null,"fr":604,"mp":0,"fp":0.00020669698222405952},{"y":1988,"t":8,"m":0,"f":8,"mr":null,"fr":969,"mp":0,"fp":0.00010854816824966079},{"y":1989,"t":17,"m":0,"f":17,"mr":null,"fr":588,"mp":0,"fp":0.0002199309158182077},{"y":1990,"t":16,"m":0,"f":16,"mr":null,"fr":626,"mp":0,"fp":0.00020382165605095542},{"y":1992,"t":8,"m":0,"f":8,"mr":null,"fr":1042,"mp":0,"fp":0.00010627274901033503},{"y":1994,"t":5,"m":0,"f":5,"mr":null,"fr":1504,"mp":0,"fp":0.0000673554887987822},{"y":1996,"t":5,"m":0,"f":5,"mr":null,"fr":1489,"mp":0,"fp":0.00006998390370214851}]