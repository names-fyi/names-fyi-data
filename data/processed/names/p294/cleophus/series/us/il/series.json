[{"y":1964,"t":6,"m":6,"f":0,"mr":691,"fr":null,"mp":0.000056807960688891204,"fp":0}]