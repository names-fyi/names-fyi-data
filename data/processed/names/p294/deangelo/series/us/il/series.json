[{"y":1965,"t":6,"m":6,"f":0,"mr":662,"fr":null,"mp":0.00006072075536619676,"fp":0},{"y":1971,"t":6,"m":6,"f":0,"mr":768,"fr":null,"mp":0.00006589785831960461,"fp":0},{"y":1972,"t":8,"m":8,"f":0,"mr":656,"fr":null,"mp":0.00009668374746205163,"fp":0},{"y":1975,"t":7,"m":7,"f":0,"mr":710,"fr":null,"mp":0.00008944658122388479,"fp":0},{"y":1976,"t":19,"m":19,"f":0,"mr":397,"fr":null,"mp":0.0002422449734168016,"fp":0},{"y":1977,"t":7,"m":7,"f":0,"mr":736,"fr":null,"mp":0.00008602890571231934,"fp":0},{"y":1978,"t":8,"m":8,"f":0,"mr":663,"fr":null,"mp":0.0001007620127212041,"fp":0},{"y":1979,"t":10,"m":10,"f":0,"mr":603,"fr":null,"mp":0.00011937875296954648,"fp":0},{"y":1980,"t":10,"m":10,"f":0,"mr":612,"fr":null,"mp":0.00011573136435705441,"fp":0},{"y":1981,"t":14,"m":14,"f":0,"mr":495,"fr":null,"mp":0.00016540056472478528,"fp":0},{"y":1982,"t":20,"m":20,"f":0,"mr":400,"fr":null,"mp":0.00023845576049503415,"fp":0},{"y":1983,"t":20,"m":20,"f":0,"mr":394,"fr":null,"mp":0.00024457053414204656,"fp":0},{"y":1984,"t":28,"m":28,"f":0,"mr":321,"fr":null,"mp":0.00034200979613773223,"fp":0},{"y":1985,"t":26,"m":26,"f":0,"mr":340,"fr":null,"mp":0.00031604025866679633,"fp":0},{"y":1986,"t":30,"m":30,"f":0,"mr":301,"fr":null,"mp":0.00037588803548383056,"fp":0},{"y":1987,"t":33,"m":33,"f":0,"mr":299,"fr":null,"mp":0.00040389699402722023,"fp":0},{"y":1988,"t":50,"m":50,"f":0,"mr":248,"fr":null,"mp":0.000599168354324198,"fp":0},{"y":1989,"t":47,"m":47,"f":0,"mr":256,"fr":null,"mp":0.0005415620030880556,"fp":0},{"y":1990,"t":48,"m":48,"f":0,"mr":264,"fr":null,"mp":0.0005380502404412012,"fp":0},{"y":1991,"t":56,"m":56,"f":0,"mr":240,"fr":null,"mp":0.000640175591019251,"fp":0},{"y":1992,"t":36,"m":36,"f":0,"mr":320,"fr":null,"mp":0.0004176043430851681,"fp":0},{"y":1993,"t":18,"m":18,"f":0,"mr":518,"fr":null,"mp":0.00021035901271503367,"fp":0},{"y":1994,"t":25,"m":25,"f":0,"mr":429,"fr":null,"mp":0.000296106787952007,"fp":0},{"y":1995,"t":28,"m":28,"f":0,"mr":408,"fr":null,"mp":0.0003389297083994045,"fp":0},{"y":1996,"t":47,"m":47,"f":0,"mr":274,"fr":null,"mp":0.0005807846771702193,"fp":0},{"y":1997,"t":35,"m":35,"f":0,"mr":338,"fr":null,"mp":0.00043948316779467345,"fp":0},{"y":1998,"t":33,"m":33,"f":0,"mr":348,"fr":null,"mp":0.0004108565737051793,"fp":0},{"y":1999,"t":33,"m":33,"f":0,"mr":359,"fr":null,"mp":0.00041355974685130647,"fp":0},{"y":2000,"t":20,"m":20,"f":0,"mr":503,"fr":null,"mp":0.0002486634340420241,"fp":0},{"y":2001,"t":24,"m":24,"f":0,"mr":443,"fr":null,"mp":0.0002991921811110003,"fp":0},{"y":2002,"t":15,"m":15,"f":0,"mr":611,"fr":null,"mp":0.0001912948108094321,"fp":0},{"y":2003,"t":12,"m":12,"f":0,"mr":722,"fr":null,"mp":0.00015147498769265726,"fp":0},{"y":2004,"t":17,"m":17,"f":0,"mr":576,"fr":null,"mp":0.00021702774125186708,"fp":0},{"y":2005,"t":19,"m":19,"f":0,"mr":549,"fr":null,"mp":0.000246299032952218,"fp":0},{"y":2006,"t":21,"m":21,"f":0,"mr":514,"fr":null,"mp":0.000272808761058498,"fp":0},{"y":2007,"t":20,"m":20,"f":0,"mr":549,"fr":null,"mp":0.00025897678273142813,"fp":0},{"y":2008,"t":20,"m":20,"f":0,"mr":545,"fr":null,"mp":0.00026760506843999626,"fp":0},{"y":2009,"t":22,"m":22,"f":0,"mr":500,"fr":null,"mp":0.0003033604059513796,"fp":0},{"y":2010,"t":17,"m":17,"f":0,"mr":610,"fr":null,"mp":0.0002435599873921889,"fp":0},{"y":2011,"t":20,"m":20,"f":0,"mr":520,"fr":null,"mp":0.00029531192321889996,"fp":0},{"y":2012,"t":24,"m":24,"f":0,"mr":463,"fr":null,"mp":0.00036101083032490973,"fp":0},{"y":2013,"t":16,"m":16,"f":0,"mr":596,"fr":null,"mp":0.00024290637477417297,"fp":0},{"y":2014,"t":15,"m":15,"f":0,"mr":675,"fr":null,"mp":0.000226053408885406,"fp":0},{"y":2015,"t":15,"m":15,"f":0,"mr":647,"fr":null,"mp":0.00022622045937834617,"fp":0},{"y":2016,"t":11,"m":11,"f":0,"mr":822,"fr":null,"mp":0.00017165774566564192,"fp":0},{"y":2017,"t":9,"m":9,"f":0,"mr":919,"fr":null,"mp":0.00014552745618006598,"fp":0},{"y":2018,"t":12,"m":12,"f":0,"mr":745,"fr":null,"mp":0.00020143015409406788,"fp":0},{"y":2019,"t":16,"m":16,"f":0,"mr":594,"fr":null,"mp":0.00027882336539802036,"fp":0},{"y":2020,"t":13,"m":13,"f":0,"mr":684,"fr":null,"mp":0.00023835716905023835,"fp":0},{"y":2021,"t":12,"m":12,"f":0,"mr":734,"fr":null,"mp":0.00022318107424490404,"fp":0},{"y":2022,"t":8,"m":8,"f":0,"mr":975,"fr":null,"mp":0.00015518913676042676,"fp":0}]