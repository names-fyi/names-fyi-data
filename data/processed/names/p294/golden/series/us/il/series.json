[{"y":1978,"t":5,"m":5,"f":0,"mr":913,"fr":null,"mp":0.00006297625795075257,"fp":0}]